modules/openapi-generator/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml

```diff
@@ -1041,6 +1041,50 @@ paths:
       responses:
         '200':
           description: Success
+  /fake/enums-of-length-one/{path_string}/{path_integer}:
```

Comment:
Endpoint added to demonstrate that enums of length one are working for an endpoint